.circleci/config.yml

```diff
@@ -1,6 +1,26 @@
 version: 2.1
+workflows:
+  version: 2
+  main:
+    jobs:
+      - debug-build-test
+      - release-build-test:
+          filters:
+            branches:
+              only: /^release\/\d+\.\d+\.\d+$/
+            tags:
+              only: /^.*$/
+      - publish-github-release:
+          requires:
+            - release-build-test
+          filters:
+            branches:
+              only: /^release\/\d+\.\d+\.\d+$/
+            tags:
+              only: /^\d+\.\d+\.\d+$/
```

Comment:
I think it is convention to prefix the tag with `v`.
My limited regex knowledge tells me that this regex will _not_ match that. Is that correct?

Reply:
This is a details that was not agreed in the spike. For me we should out the v everywhere or nowhere.

Reply:
`comit-i` uses `vX.Y.Z` tags and GitHub suggests that as-well when creating a tag. I'd vote for `v`-style tags.

Reply:
Ok we agreed on this one :)